tv-shows/src/components/features/shows/ShowPicker/components/ShowPickerProgress.tsx

```diff
@@ -0,0 +1,19 @@
+'use client';
+
+import { Progress } from '@chakra-ui/react';
+import { useContext } from 'react';
+import { ShowPickerContext } from './ShowPickerContextProvider';
+
+export const ShowPickerProgress = () => {
```

Comment:
![image](https://github.com/user-attachments/assets/a7b11106-29c5-4bab-98b6-dc0b5bc17e30)
Progress component tells me I'm done here, but I don't see the results.

Reply:
Offset by one